src/utils/advancedSearchUtils.ts

```diff
@@ -0,0 +1,38 @@
+import type { SearchParams, SearchFormInputField } from "../types/searchTypes"
+import { searchAggregations } from "../config/aggregations"
+
+export const textInputFields: SearchFormInputField[] = [
+  { name: "q", key: "searchKeywords", label: "Keywords" },
```

Comment:
I think this creates an issue in `advanced.tsx` line 133 where the value is `searchFormState[name]`. For this first object, the name is `"q"` but that doesn't exist in the `initialSearchFormState` object.

Reply:
I think making that name consistent will fix this issue:
<img width="1324" alt="Screen Shot 2023-09-20 at 11 47 32 AM" src="https://github.com/NYPL/research-catalog/assets/1280564/3c30d9ea-a76c-4c99-9bf7-f261cc5d91c0">

Reply:
I might actually rename serachKeywords to q in SearchParams for consistency. This will eliminate the need for the key lookup.